src/Settings/RTK.SettingsGroup.json

```diff
@@ -20,5 +20,55 @@
     "units":                "secs",
     "decimalPlaces":        0,
     "qgcRebootRequired":    true
+},
+{
+    "name":                 "UseFixedBasePosition",
+    "shortDescription":     "Use specified base position",
+    "longDescription":      "Specify the values for the RTK base position without having to do a survey in.",
+    "type":                 "bool",
+    "defaultValue":         false,
+    "qgcRebootRequired":    true
+},
+{
+    "name":                 "FixedBasePositionLatitude",
+    "shortDescription":     "Base Position Latitude",
+    "longDescription":      "Defines the latitude of the fixed RTK base position.",
+    "type":                 "double",
+    "defaultValue":         0,
+    "min":                  -90,
+    "max":                  90,
+    "decimalPlaces":        7,
```

Comment:
Can you increase this to 9? That's what's used internally.

Reply:
Ah, thanks. I missed this before I posted my above decimal place question.